cpp/trtorchc/main.cpp

```diff
@@ -1,14 +1,21 @@
 #include <iostream>
 #include <sstream>
 #include <stdlib.h>
-#include <unistd.h>
 
 #ifdef linux
 #include <linux/limits.h>
 #else
 #define PATH_MAX 260
 #endif
 
+#ifdef _WIN32
```

Comment:
Can you please use #if defined(_WIN32) instead?

Reply:
I was copying the styling from above where #ifdef linux is used.

Reply:
@narendasan : We should change rest of the #ifdef and #ifndef